tachyon/__main__.py

```diff
@@ -113,20 +109,20 @@ def load_execute_file_plugins():
 
 def print_program_header():
     """ Print a _cute_ program header """
-    print("\n\t Tachyon v" + conf.version + " - Fast Multi-Threaded Web Discovery Tool")
-    print("\t https://github.com/delvelabs/tachyon\n")
+    return "\n\t Tachyon v" + __version__ + " - Fast Multi-Threaded Web Discovery Tool\n" \
+                                            "\t https://github.com/delvelabs/tachyon\n"
 
 
-async def scan(hammertime, *, directories_only=False, files_only=False, plugins_only=False):
-    if conf.cookies is not None:
-        set_cookies(hammertime, conf.cookies)
+async def scan(hammertime, *, cookies=None, directories_only=False, files_only=False, plugins_only=False, **kwargs):
+    if cookies is not None:
+        set_cookies(hammertime, cookies)
     else:
         await get_session_cookies(hammertime)
 
-    load_execute_host_plugins()
+    #load_execute_host_plugins()
```

Comment:
This seems like a mistake.

Reply:
plugins rely on configuration in conf.py and no longer work now. Plugin migration will be done in the future.